sdk/python/approzium/iam.py

```diff
@@ -1,13 +1,25 @@
 import boto3
 
 
-def obtain_signed_get_caller_identity(iam_role):
+def assume_role(iam_role):
     if iam_role is None:
         raise NotImplementedError("Automatic IAM ARN determination is not implemented")
     sts_client = boto3.client("sts")
```

Comment:
I think with the changes you made, it should be straightforward to implement this. Could be either part of this PR or a later one.

Reply:
Not sure what you're referring to. Can you elaborate?

Reply:
Going to do in a separate PR.